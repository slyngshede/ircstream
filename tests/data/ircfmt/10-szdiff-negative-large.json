{
  "$schema": "/mediawiki/recentchange/1.0.0",
  "meta": {
    "uri": "https://www.wikidata.org/wiki/Q87697904",
    "request_id": "660e7ac1-bc30-4d73-83c5-30d688761496",
    "id": "342abd5f-95f1-496b-9f9f-c26738f51708",
    "dt": "2024-10-01T22:08:18Z",
    "domain": "www.wikidata.org",
    "stream": "mediawiki.recentchange",
    "topic": "codfw.mediawiki.recentchange",
    "partition": 0,
    "offset": 1136213542
  },
  "id": 2322341971,
  "type": "edit",
  "namespace": 0,
  "title": "Q87697904",
  "title_url": "https://www.wikidata.org/wiki/Q87697904",
  "comment": "/* wbeditentity-update-languages-and-other-short:0||gl, mul, an, ia, en, oc, pt, la, ca, io, ast, vo, pt-br, fr, nl, eu, ga, ar, de, ext, dv, pl, es, it, ie, eo */ mv labels and aliases matching [[Property:P528]] or [[Property:P3083]] to mul",
  "timestamp": 1727820498,
  "user": "Twofivesixbot",
  "bot": true,
  "notify_url": "https://www.wikidata.org/w/index.php?diff=2255252688&oldid=2164113324&rcid=2322341971",
  "minor": false,
  "patrolled": true,
  "length": {
    "old": 25901,
    "new": 24116
  },
  "revision": {
    "old": 2164113324,
    "new": 2255252688
  },
  "server_url": "https://www.wikidata.org",
  "server_name": "www.wikidata.org",
  "server_script_path": "/w",
  "wiki": "wikidatawiki",
  "parsedcomment": "‎<span dir=\"auto\"><span class=\"autocomment\">Changed label, description and/or aliases in gl, mul, an, ia, en, oc, pt, la, ca, io, ast, vo, pt-br, fr, nl, eu, ga, ar, de, ext, dv, pl, es, it, ie, eo, and other parts: </span></span> mv labels and aliases matching <a href=\"/wiki/Property:P528\" title=\"‎catalog code‎ | ‎catalog name of an object, use with qualifier P972‎\"><span class=\"wb-itemlink\"><span class=\"wb-itemlink-label\" lang=\"en\" dir=\"ltr\">catalog code</span> <span class=\"wb-itemlink-id\">(P528)</span></span></a> or <a href=\"/wiki/Property:P3083\" title=\"‎SIMBAD ID‎ | ‎identifier for an astronomical object, in the University of Strasbourg&#039;s SIMBAD database‎\"><span class=\"wb-itemlink\"><span class=\"wb-itemlink-label\" lang=\"en\" dir=\"ltr\">SIMBAD ID</span> <span class=\"wb-itemlink-id\">(P3083)</span></span></a> to mul"
}
