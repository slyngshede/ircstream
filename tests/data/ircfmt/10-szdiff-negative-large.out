#wikidata.wikipedia
14[[07Q8769790414]]4 B10 02https://www.wikidata.org/w/index.php?diff=2255252688&oldid=2164113324&rcid=2322341971 5* 03Twofivesixbot 5* (-1785) 10/* wbeditentity-update-languages-and-other-short:0||gl, mul, an, ia, en, oc, pt, la, ca, io, ast, vo, pt-br, fr, nl, eu, ga, ar, de, ext, dv, pl, es, it, ie, eo */ mv labels and aliases matching [[Property:P528]] or [[Property:P3083]] to mul
